$OpenBSD$

Fix warning: initialization from incompatible pointer type
Due to E_Gadcon_Client_Class change

--- src/modules/backlight/e_mod_main.c.orig	Sun Oct 16 18:31:29 2011
+++ src/modules/backlight/e_mod_main.c	Wed Nov  2 21:31:46 2011
@@ -5,7 +5,7 @@
 static E_Gadcon_Client *_gc_init(E_Gadcon *gc, const char *name, const char *id, const char *style);
 static void _gc_shutdown(E_Gadcon_Client *gcc);
 static void _gc_orient(E_Gadcon_Client *gcc, E_Gadcon_Orient orient);
-static char *_gc_label(E_Gadcon_Client_Class *client_class);
+static const char *_gc_label(E_Gadcon_Client_Class *client_class);
 static Evas_Object *_gc_icon(E_Gadcon_Client_Class *client_class, Evas *evas);
 static const char *_gc_id_new(E_Gadcon_Client_Class *client_class);
 
@@ -437,7 +437,7 @@ _gc_orient(E_Gadcon_Client *gcc, E_Gadcon_Orient orien
    e_gadcon_client_min_size_set(gcc, mw, mh);
 }
 
-static char *
+static const char *
 _gc_label(E_Gadcon_Client_Class *client_class __UNUSED__)
 {
    return _("Backlight");
